packages/wdio-concise-reporter/tests/index.test.ts

```diff
@@ -20,14 +19,18 @@ describe('ConciseReporter', () => {
 
     describe('on create', () => {
         it('should verify initial properties', () => {
-            expect(Array.isArray(reporter.suiteUids)).toBe(true)
-            expect(reporter.suiteUids.length).toBe(0)
-            expect(Array.isArray(reporter.suites)).toBe(true)
-            expect(reporter.suites.length).toBe(0)
-            expect(reporter.stateCounts).toEqual({
+            // @ts-ignore
+            expect(Array.isArray(reporter._suiteUids)).toBe(true)
```

Comment:
You can remove the ignore comments by accessing the private properties in a different way:
```suggestion
            expect(Array.isArray(reporter['_suiteUids'])).toBe(true)
```